src/xrCore/FS.cpp

```diff
@@ -88,6 +88,9 @@ void VerifyPath(LPCSTR path)
         tmp[i] = 0;
         _mkdir(tmp);
     }
+#ifdef LINUX
+    while (char* sep = strchr((char *)path, '\\')) *sep = '/';
```

Comment:
Please consider moving this code (slashes conversion) to separate function (named like ConvertGamePathToPlatformSpecificPath) with platform-specific implementation. It makes the code more clear and allows to reduce code duplication.